File: Chapter-03/Array-2nd-to-Last/Array-2nd-to-Last.py
# Chapter-3: Arrays
# Array-2nd-to-Last
# Return the 2nd to last element of an array. Given [1,2,-3,4] return -3. If array is too short return null

# Assume the argument passed is an array
def secondToLast(arr):
    if len(arr) < 2:
        return None
    else:
        return arr[len(arr) - 2]

myArr = [6]
print("The second to last element in {} is {}").format(myArr, secondToLast(myArr))

File: Chapter-03-Arrays/Array-2nd-to-Last/Array-2nd-to-Last.js
// Chapter-3: Arrays
// Array-2nd-to-Last
// Return the 2nd to last element of an array. Given [1,2,-3,4] return -3. If array is too short return null

// Assume the argument passed is an array
function secondToLast(arr){
  if (arr.length < 2){
    return null
  } else {
    return arr[arr.length - 2]
  }
}

var myArr = [0,6]
console.log(`The second to last element in ${myArr} is ${secondToLast(myArr)}`)
